### ISSUE TITLE
Co-Processor Unit not splitting the job workload

### ISSUE THREAD

**Issue by FelippeChemello** [AppliedEnergistics/Applied-Energistics-2#6790]
### Describe the bug

When trying to auto craft a multi step crafting, with multiple CPUs and co-processor unit, it doesn't split the job into various CPUs making all steps sequential.
An example is at crafting Insanium Essence at ATM8, I'd setted all crafts and put each patter in one pattern provider with one molecular assembler and inserted in the system 5 Master Infusion Crystal (that is needed to craft simultaneos the 5 steps of craft), however as you can seed on the provided screenshots it is crafting one recipe after another instead of splitting each recipe to each CPU.

### How to reproduce the bug

- Add Crafting CPUs with co-processor unit (same quantity as the steps of recipe)
- Set all the patterns of recipe
- Ask to craft the recipe

### Expected behavior

Split each step of recipe in each CPU to speed up crafting time.

### Which minecraft version are you using?

1.19

### On which mod loaders does it happen?

Forge

### Crash log

https://pastebin.com/hVTVNKnQ

**Comment by Technici4n** [comment]
I don't see any screenshot, but are you correctly using many molecular assemblers? Because the CPUs with co-processors dispatch things faster, but they are still limited by the speed at which the molecular assemblers are crafting.

**Comment by Technici4n** [comment]
Ah, I saw your screenshots on another issue.

That's not how co-processors work. They only make their CPU faster, but each crafting job is only assigned to one CPU. Make sure to use multiple full speed molecular assemblers around your pattern providers for optimal crafting performance. You can even put the same pattern in multiple pattern providers to have them craft in parallel.

**Comment by Mari023** [comment]
the actual problem is the cpu not taking all 5 master infusion crystals, and therefor it can't run the recipes in parallel

**Comment by FelippeChemello** [comment]
Sorry @Technici4n  I messed up, I was checking another issue and added the screenshots there, instead here. Sorry.
Yes, there is the bug @Mari023 , as I've seen in some gameplays, it should uses all master infusion crystal at same time, speeding up the craft, however I was not able to do that. 
Eg. I ask to craft 10 Insanium and had 36 supremium (that can make 9 insanium) on the drivers, as I understood from the docs, it would craft at same time 9 Insanium in one CPU while other one crafts all the tiers needed to completed the 10th Insanium, however it doesn't happen, even if I put 6 molecular assembler in each pattern provider it crafts just one recipe per time.

**Comment by Technici4n** [comment]
Ah yes I see. Sadly this is not fixable. The system tries to be smart and reuse the crystals, but there is no way to ask for a specific amount to be not be reused, as the simulation and the execution happen separately. A workaround is to use a crafter from another mod

**Comment by EnderProyects** [comment]
You can also make the craft in a subnet, so one crystal per subnet at the same time
